turbinia/workers/__init__.py

```diff
@@ -174,7 +168,7 @@ def close(self, task, success, status=None):
           task.output_manager.save_evidence(evidence, self)
       else:
         self.log(
-            'Evidence {0:s} has empty or missing file at source_path {1:s} so '
+            'Evidence {0!s:s} has empty or missing file at source_path {1!s:s} so '
```

Comment:
I think this is what you want here:
```suggestion
            'Evidence {0!s} has empty or missing file at source_path {1!s} so '
```